lib/matplotlib/backends/backend_qt5.py

```diff
@@ -415,9 +415,6 @@ def stop_event_loop(self):
 
     stop_event_loop.__doc__ = FigureCanvasBase.stop_event_loop_default.__doc__
 
-    def draw_idle(self):
```

Comment:
This should probably stay, do we want to fallback to the base implementation of `draw_idle`?